pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -40,15 +40,22 @@ const (
 	checkSleepDuration = time.Second
 	diskByIDPath       = "/dev/disk/by-id/"
 	diskSCSIPrefix     = "wwn-0x"
-	diskformat         = "diskformat"
-	datastore          = "datastore"
-	StoragePolicyName  = "storagepolicyname"
+	// diskformat parameter is deprecated as of Kubernetes v1.21.0
```

Comment:
After the deprecation, will the default disk format will continue to be "thin"?

Reply:
After deprecation, if the user has enabled vSphere CSI Migration,  and if `thick` or `zeoredthick` value is specified for `diskformat` parameter, new volume creation using the in-tree vsphere storage class will fail.

If migration is not enabled, the `diskformat` parameter will continue to work as it is.